src/main/java/com/google/sps/data/GovPostReq.java

```diff
@@ -34,12 +49,15 @@
     //Enable DoOutput to send request content and write content to output stream
     con.setDoOutput(true);
 
+    String desiredDate = inputDate.format(DateTimeFormatter.ofPattern("yyyy-MM-dd"));
+    System.out.println(LocalDate.now());
```

Comment:
Try to remove debug output before requesting merges.

Reply:
this file is deleted in the latest push